build.psm1

```diff
@@ -458,10 +458,11 @@ function Start-PSBuild {
     }
 
     if ($Clean) {
-        Write-Log "Cleaning your working directory. You can also do it with 'git clean -fdX'"
+        Write-Log "Cleaning your working directory. You can also do it with 'git clean -fdx --exclude .vs/PowerShell/v15/Server/sqlite3'"
         Push-Location $PSScriptRoot
         try {
-            git clean -fdX
+            # Excluded sqlite3 folder is due to this Roslyn issue: https://github.com/dotnet/roslyn/issues/23060
+            git clean -fdx --exclude .vs/PowerShell/v15/Server/sqlite3
             # Extra cleaning is required to delete the CMake temporary files.
             # These are not cleaned when using "X" and cause CMake to retain state, leading to
```

Comment:
maybe update the comment

Reply:
Should we rather remove this block and the 2nd `git clean -gdx` below?
Was there a reason why the -X option was chosen initially?

Reply:
I'm not sure.  The results of `-gdx` seem better to me with no unexpected effects.